Resources/translations/messages.en.xliff

```diff
@@ -198,6 +198,14 @@
                 <source>No result</source>
                 <target>No result</target>
             </trans-unit>
+            <trans-unit id="45">
+                <source>Selected _s_ rows</source>
```

Comment:
What is this `_s` here?

Reply:
See this: https://github.com/APY/APYDataGridBundle/blob/58f4e370ba79bd7d593a3759010ab002b3946b61/Resources/views/blocks.html.twig#L505

Reply:
Ok, got it